pynxtools/dataconverter/logger.py

```diff
@@ -22,4 +22,8 @@
 logger = logging.getLogger("pynxtools")
 
 # Lowest level log allows to other levels erros, crittical, info and debug
+handler = logging.StreamHandler()
+formatter = logging.Formatter("%(message)s")
+handler.setFormatter(formatter)
+logger.addHandler(handler)
```

Comment:
I think some code, imports maybe, are setting the basic loglevel to warning again. If I set logging.basicconfig(level=debug) it works again.

Will these changes here affect Nomad's log output?

Reply:
If I understand correctly, this should only effect the pynxtools level logger, but I am not sure.